ql/src/java/org/apache/hadoop/hive/ql/exec/vector/mapjoin/fast/VectorMapJoinFastHashTableLoader.java

```diff
@@ -312,13 +312,11 @@ public void load(MapJoinTableContainer[] mapJoinTables,
                   inputName, cacheKey, receivedEntries, delta);
         }
       } catch (InterruptedException e) {
+        loadExecService.shutdownNow();
```

Comment:
should this be `finally`?

Reply:
Good catch! Thanks for finding this. Addressed!